ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml

```diff
@@ -66,6 +66,9 @@ ref:
   - name: PRIVATE_LINK
     default: "false"
     documentation: Provides private connectivity between VPCs, AWS services, and your on-premises networks, without exposing your traffic to the public internet.
+  - name: USER_TAGS
```

Comment:
Let us use `CLUSTER_TAGS` instead of `USER_TAGS`. And the default value is set to `cluster:${CLUSTER_NAME}`.

Reply:
Ack, I will switch to CLUSTER_TAGS.
I want to ensure the default is set to `ciprow:{CLUSTER_NAME}` allowing users to understand
this tag is coming from ci-prow.

Reply:
How about the tag `prowci:true`?

Reply:
`prowci:{CLUSTER_NAME}` having a unique value allows us to track our cloud-expenses.
So, would like to stick with this `prowci:{CLUSTER_NAME}`